session/payment/session_payments_test.go

```diff
@@ -134,3 +133,12 @@ func Test_SessionPayments_ErrsOnBalanceMissmatch(t *testing.T) {
 	balanceChannel <- balance.Message{Balance: 100, SequenceID: 1}
 	<-testDone
 }
+
+func Test_SessionPayments_NoPanicOnSecondStop(t *testing.T) {
+	balanceChannel := make(chan balance.Message, 1)
+	promiseSender := newPromiseSender()
+	cpo := NewTestSessionPayments(balanceChannel, promiseSender, promiseTracker, balanceTracker)
+	go cpo.Start()
+	cpo.Stop()
+	cpo.Stop()
```

Comment:
does this tests what it is suppose to test? `go cpo.Start()` does not guarantee that session has been started by the time `cpo.Stop()` executes.

Reply:
It tests `close of closed channel`, the channel created at `NewTestSessionPayments` and closed at `Stop()`.

Reply:
Then we probably don't need `go cpo.Start()` at all in test?